v2/brigadier/src/workers.ts

```diff
@@ -1,7 +1,20 @@
+/**
+ * Represents the Brigade worker within which an event handler runs.
+ */
 export interface Worker {
+  /** The address of the Brigade API server. */
   apiAddress: string
+  /** A token which can be used to authenticate to the API server. */
   apiToken: string
+  /**
+   * The directory where the worker stores configuration files,
+   * including event handler code files such as brigade.js and brigade.json.
+   */
```

Comment:
This defaults to `./.brigade`

Reply:
But maybe not worth mentioning. `Worker` isn't something the user instantiates and configures. It's something that's handed _to_ them as part of the event they receive.

Forget I said anything.

Reply:
Yeah we should really make this interface `readonly`, not that it would stop JavaScript consumers.